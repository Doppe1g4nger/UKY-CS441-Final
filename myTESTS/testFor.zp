int main()
{
    int i;
    i=0;
    for(i < 5; i = i + 1)
    {
        int var;
        var = getnum();
        if (var < 50)
        {
            puts("Less than 50\n");
        }
    }
    return 0;
}
