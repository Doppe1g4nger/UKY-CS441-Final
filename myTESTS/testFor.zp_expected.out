Less than 50
Less than 50
Less than 50
Less than 50
Less than 50
